blink.js

```diff
@@ -1 +1,19 @@
-// YOUR CODE GOES HERE
+//Note to self. More info on plugin creation at https://learn.jquery.com/plugins/basic-plugin-creation/
+(function($)
+{
+  $.fn.blink = function(options) {
+    return this.each(function() {
+      var obj = $(this);
+      setInterval(function() {
+        if($(obj).css("visibility") == "visible")
+        {
+          $(obj).css('visibility','hidden');
+        }
+        else
+        {
+          $(obj).css('visibility','visible');
+        }
+      }, options);
```

Comment:
`options` usually means it's an object with zero or more properties...what would you call this instead to make it more clear?